tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -658,6 +631,10 @@ class MklDummyOp : public OpKernel {
   }
 };
 
+#ifdef ENABLE_MKLDNN_V1
+#undef MKLDNN_SIZE_DTYPE
```

Comment:
`MKLDNN_SIZE_DTYPE` is always defined, we don't need to guard it with v1 ifdef.

Reply:
Yes, you're right :) Fixed it.